package.json

```diff
@@ -100,7 +100,7 @@
 		"eslint-branch": "check-npm-client && node bin/eslint-branch.js",
 		"install-if-deps-outdated": "check-npm-client && node bin/install-if-deps-outdated.js",
 		"install-if-no-packages": "check-npm-client && node bin/install-if-no-packages.js",
-		"postinstall": "check-npm-client && (node -e \"process.exit(process.env.CALYPSO_DOCTOR_SKIP !== 'true')\" || yarn run calypso-doctor) && yarn run build-packages",
+		"postinstall": "test -n \"$DISABLEPOSTINSTALL\" || (check-npm-client && (node -e \"process.exit(process.env.CALYPSO_DOCTOR_SKIP !== 'true')\" || yarn run calypso-doctor) && yarn run build-packages)",
```

Comment:
How much time is this actually saving? 

I'd consider moving `build-packages` out of here (maybe run it before `yarn start`, but not on `postinstall`?) before adding the complexity of skipping `build-packages` in some cases

Reply:
On a `machine` executor, impact to the node module installation (`Install Dependencies`) step is drastic.

- Cold cache: 6m 36s ([link](https://app.circleci.com/pipelines/github/Automattic/wp-calypso/69869/workflows/0ae2e9c4-8323-4824-b532-3ef4c621c242/jobs/843787))
- Warm cache: 2m 7s ([link](https://app.circleci.com/pipelines/github/Automattic/wp-calypso/69873/workflows/6cd21a0a-ac17-4e77-b873-d66fcb29a934/jobs/843853))

> consider moving build-packages out of here (maybe run it before yarn start, but not on postinstall

That's certainly an idea. However I'd be concerned that other Calypso scripts in this Circle config depend on the existing behavior (I can't see `yarn start` being invoked anywhere).

Reply:
I see. In the logs of the first link, `yarn run build-packages` takes almost three minutes (`"lerna success run Ran npm script 'prepare' in 23 packages in 166.8s:"`).

The thing is that, in theory, you have to run those anyway at some point because `webpack` expects those packages to be already transpiled. I'm very surprised that you can skip building packages and it doesn't fail later.

Reply:
> I'm very surprised that you can skip building packages and it doesn't fail later.

We're restoring the entire built `packages` directory, if it's available, so we're not really skipping the building step, yeah?

Reply:
Ah right, forgot that part 🤦